packages/flutter/lib/src/widgets/scroll_position_with_single_context.dart

```diff
@@ -197,7 +197,7 @@ class ScrollPositionWithSingleContext extends ScrollPosition implements ScrollAc
     goIdle();
     if (pixels != value) {
       final double oldPixels = pixels;
-      forcePixels(value);
+      correctPixels(value);
       notifyListeners();
```

Comment:
Current:
```dart
      correctPixels(value);
      notifyListeners();
```

Suggest:
```suggestion
      forcePixels(value)
```

After reading the documentation for `correctPixels` and `forcePixels`, I think this should be the other way around, removing the call to `notifyListeners` in `jumpTo` and keeping the `forcePixels` call.